kfdef/README.md

```diff
@@ -0,0 +1,36 @@
+# Deploy latest DSPO via ODH 
+
+To deploy the latest DSPO using the changes within this repo via Open Data Hub you can follow these steps
+
+
+## Pre-requisites
+1. An OpenShift cluster that is 4.10 or higher.
+2. You will need to be logged into this cluster as [cluster admin] via [oc client].
+3. The OpenShift Cluster must have OpenShift Pipelines 1.9 or higher installed. Instructions [here][OCP Pipelines Operator].
+4. The Open Data Hub operator needs to be installed. You can install it via [OperatorHub][installodh].
+
+
+## Deploy Kfdef
+
+Clone this repository then run the following commands: 
+
+```bash 
+git clone https://github.com/opendatahub-io/data-science-pipelines-operator.git 
+cd data-science-pipelines-operator/kfdef 
+
+# If this namespace does not exist
+oc new-project odh-applications 
+
+# Then run
+oc -n odh-applications apply -f kfdef.yaml
```

Comment:
I recently learned that you can pass a URL to `oc apply`, so this command could be:
```
oc apply -f https://raw.githubusercontent.com/HumairAK/data-science-pipelines-operator/add_kfdef/kfdef/kfdef.yaml -n odh-applications
```

If we document it that way then we can eliminate the `git clone` and `cd` steps above.

Reply:
done